### DISCUSSION TITLE
Welcome to ligolo-ng Discussions!

### DISCUSSION THREAD

**Discussion by nicocha30** [nicocha30/ligolo-ng#90] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by FazmaFax** [comment]
I'd like to know if this is something I could download on a raspberry pi, and if so, what file should I download?

**Reply by nicocha30** [reply]
https://docs.ligolo.ng

Use the ARM64 build.
